rueppellii-lvlup-asp.net-core/Controllers/PitchController.cs

```diff
@@ -37,7 +38,7 @@ public IActionResult GetPitches()
                 return StatusCode(401, new ErrorMessage("Unauthorizied"));
             }
 
-            return Ok("success");
+            return Ok(DummyJsonResponseDTO.json);
```

Comment:
the naming .json is not telling too much, it would be more readable to call it as getpitches or something like that

Reply:
but good job, you are learning quick! :)